meson.build

```
@@ -2,9 +2,11 @@ project('seldon', 'cpp',
   version : '0.1',
   default_options : ['warning_level=3', 'cpp_std=c++20', 'optimization=3'])
 
-add_global_arguments(['-Wno-unused-local-typedefs', '-Wno-array-bounds'], language : 'cpp')
+deps = []
+cpp_args = ['-Wno-unused-local-typedefs', '-Wno-array-bounds']
 
 incdir = include_directories('include')
+deps += [dependency('fmt'), dependency('tomlplusplus')]
```

```suggestion
_incdir = include_directories('include')
_deps += [dependency('fmt'), dependency('tomlplusplus')]
```

I have a preference for using _ for scoping variables, otherwise it gets a bit confusing in larger projects